pi_ldapproxy/proxy.py

```diff
@@ -3,15 +3,16 @@
 import json
 import sys
 import re
-import urllib
-from cStringIO import StringIO
+import urllib.request, urllib.parse, urllib.error
+from io import BytesIO
 from functools import partial
 
 from ldaptor.protocols import pureldap
 from ldaptor.protocols.ldap import ldaperrors
 from ldaptor.protocols.ldap.ldapclient import LDAPClient
 from ldaptor.protocols.ldap.ldapconnector import connectToLDAPEndpoint
 from ldaptor.protocols.ldap.proxybase import ProxyBase
+from six import ensure_str
```

Comment:
This does not work out - probably with the six-version we are using in requirements.txt?
We should then also update six to 1.16  in the requirements.txt

~~~
builtins.ImportError: cannot import name 'ensure_str' from 'six' (.../ldapproxy/lib/python3.8/site-packages/six.py)
~~~

Reply:
OK, I will update the reqs on my own.